React/src/App.tsx

```diff
@@ -1,17 +1,87 @@
-import React, { useCallback, useState } from 'react';
+import React, {
+  useRef, useMemo, useCallback, RefObject,
+} from 'react';
 import './App.css';
 import 'devextreme/dist/css/dx.material.blue.light.compact.css';
-import Button from 'devextreme-react/button';
+
+import Scheduler, { Resource } from 'devextreme-react/scheduler';
+import { ClickEvent } from 'devextreme/ui/button';
+import { AppointmentTooltipTemplateData } from 'devextreme/ui/scheduler';
+
+import { employees, data, Employee } from './data';
+import DataCell from './DataCell';
+import ResourceCell from './ResourceCell';
+import Tooltip from './Tooltip';
+
+const currentDate = new Date(2021, 5, 2, 11, 30);
+const groups = ['employeeID'];
+const views = ['month'] as ['month'];
+
+function getDisabled(employeeID: number): boolean | undefined {
```

Comment:
Question.

Why do we not use the arrow functions in this example?

Reply:
Linter required functions for some reasons:
![image](https://github.com/DevExpress-Examples/devextreme-scheduler-appointment-template/assets/111366900/7c8cc419-289b-4071-99f8-ae98ccc390d3)